diff --git a/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java b/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
index 3e9a4e2..282c851 100644
--- a/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
+++ b/iMage/jmjrst.main/src/main/java/org/jis/generator/LayoutGalerie.java
@@ -548,15 +548,24 @@
 
   public void copyFile(File file, File ziel) throws FileNotFoundException, IOException
   {
-    in = new BufferedInputStream(new FileInputStream(file));
-    out = new BufferedOutputStream(new FileOutputStream(ziel, true));
-    int bytes = 0;
-    while ((bytes = in.read()) != -1)
-    {
-      out.write(bytes);
-    }
-    in.close();
-    out.close();
+	  try {
+		  in = new BufferedInputStream(new FileInputStream(file));
+		    out = new BufferedOutputStream(new FileOutputStream(ziel, false));
+		    int bytes = 0;
+		    while ((bytes = in.read()) != -1)
+		    {
+		      out.write(bytes);
+		    }
+	  } finally {
+		  if (in != null) {
+			  in.close();
+		  }
+		  if (out != null) {
+			  out.close();
+		  }
+	  }
+    
+    
   }
 
   private String createTitle(String filename)
